packages/api/src/cors.js

```diff
@@ -22,7 +22,7 @@ export function corsOptions (request) {
       // such as Authorization (Bearer) or X-Client-Name-Version
       'Access-Control-Allow-Headers':
         headers.get('Access-Control-Request-Headers') || '',
-      'Access-Control-Expose-Headers': 'Link'
+      'Access-Control-Expose-Headers': 'Link, Count, Page, Size'
```

Comment:
These are not standard HTTP headers to my knowledge.

Reply:
Nope, these are custom.

I think we could instead use [`range` ](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Range). 

Alternatively we could include pagination meta in the response body, this would be a breaking change so deliberately avoided this in favour of using the headers for now.